.gitignore

```diff
@@ -1,3 +1,14 @@
+*.iml
```

Comment:
Use a global .gitignore file instead of upload this to the repo. It depends a lot of the IDE that you usually use.